cluster/gce/upgrade.sh

```diff
@@ -430,7 +444,7 @@ function do-node-upgrade() {
 
   # Remove the old templates.
   echo "== Deleting old templates in ${PROJECT}. ==" >&2
-  for tmpl in ${old_templates[@]}; do
+  for tmpl in "${old_templates[@]}"; do
```

Comment:
are you _sure_ this works exactly the same as before? I think we actually want globbing/splitting here